styles/feed.js

```diff
@@ -183,7 +183,7 @@ export const commentFormClass = {
     form: 'flex flex-col',
     body: 'inline-flex items-center space-x-md',
     commentBar:
-        'inline-flex rounded-full items-center py-1 px-2 md:px-5 lg:w-96 xl:w-96 \
+        'inline-flex rounded-[10px] items-center py-2 px-2 md:px-5 lg:w-96 xl:w-96 \
```

Comment:
Please change this line to:
`inline-flex rounded-[10px] items-center py-2 pl-2 pr-1 md:pl-5 md:pr-3 lg:w-96 xl:w-96`

so the px-2 and md:px-5 are decomposed into different left and right margins.